tools/recetox_aplcms/recetox_aplcms_remove_noise.xml

```diff
@@ -0,0 +1,59 @@
+<tool id="recetox_aplcms_remove_noise" name="RECETOX apLCMS - remove noise" version="@TOOL_VERSION@+galaxy0">
+    <description>filter noise and detect peaks from LC/MS data</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator"/>
+    <expand macro="requirements">
+        <requirement type="package" version="2.5.2">pymzml</requirement>
```

Comment:
Good point. I guess there is no need to have two separate containers for some steps of the same tool. If anything, I suspect it can create confusion if it installs different versions of some unpinned dependencies in different containers.

Reply:
Unified in [8dae925](https://github.com/RECETOX/galaxytools/pull/317/commits/8dae9254ae5e02962cc504417d0cc81cf4730f6f).